frontend/src/pages/NewRunV2.tsx

```diff
@@ -327,13 +264,9 @@ function NewRunV2(props: NewRunV2Props) {
         )}
 
         {/* Pipeline selection */}
-        {(pipelineId || (apiResourceRefFromRun && apiResourceRefFromRun[1])) && (
+        {apiPipeline && (
```

Comment:
What happen if `apiPipelineVersion` exist but `apiPipeline` doesn't exist?
Can we have a wrapper component which say:

```
{apiPipeline && apiPipelineVersion && (
  
        {/* Pipeline selection */}

       ......
  

        {/* Pipeline version selection */}


         .....
)}

Then remove the individual check in this line and below?

Reply:
> What happen if `apiPipelineVersion` exist but `apiPipeline` doesn't exist?
I thought to only show apiPipelineVersion.